src/components/about-section.css

```diff
@@ -0,0 +1,43 @@
+.about-section h2 {
+    font-family: "Signika", sans-serif;
+    font-size: xx-large;
+    margin-top: 2rem;
+  }
+  
+  .about-section p {
+    font-family: "Open Sans", sans-serif;
+    line-height: 1.4;
+    margin-bottom: 3rem;
+  }
+  
+  .about-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: white;
+    border-top-left-radius: 2rem;
```

Comment:
Remove this line

Reply:
was not able to remove the lines because it affected the colour and position of the paragraph

Reply:
It was only the lines `border-top-left-radius` and `transform` which needed to be removed.
Please add back `align-items: center;`

Reply:
done